### PR TITLE
Added new cadastral number provider

### PR DESCRIPTION
**Pull request by eldario**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes

## Description

`CadastralNumberProvider` faker provider for random cadastral numbers

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code
- [x] I have made changes in [CHANGELOG.md](https://github.com/avto-dev/faker-providers/blob/master/CHANGELOG.md) file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,6 +192,16 @@ $faker->validKppCode(); // 773301001 7733AZ001
 $faker->invalidKppCode(); // 7733010011 77330100Z
 ```
 
+### `AvtoDev\FakerProviders\Providers\Identifiers\CadastralNumberProvider`
+
+```php
+<?php /** @var \Faker\Generator|\AvtoDev\FakerProviders\ExtendedFaker $faker */
+
+$faker->cadastralNumber(); // 66:41:153222:68
+$faker->validCadastralNumber(); // 77:22:5874698:1
```
Comment (tarampampam) [inline README.md:201]: Are you sure about this value is really valid?

File: src/Frameworks/Laravel/ServiceProvider.php [hunk 2]
```diff
@@ -108,6 +108,7 @@ protected function packageProviders(): array
             \AvtoDev\FakerProviders\Providers\Packages\IDEntityProvider::class,
             \AvtoDev\FakerProviders\Providers\User\AvatarUriProvider::class,
             \AvtoDev\FakerProviders\Providers\Identifiers\InnAndKppProvider::class,
+            \AvtoDev\FakerProviders\Providers\Identifiers\CadastralNumberProvider::class,
```
Comment (tarampampam) [inline src/Frameworks/Laravel/ServiceProvider.php:111]: Write (update) test for this line

File: README.md [hunk 3]
```diff
@@ -59,6 +59,7 @@ $ php artisan vendor:publish --provider="AvtoDev\FakerProviders\Frameworks\Larav
 - Объект IDEntity (необходима установка дополнительного пакета [avto-dev/identity-laravel][identity]).
 - Ссылку на аватар пользователя
 - ИНН и КПП организации
+- Кадастровый номер объекта недвижимости
```
Comment (tarampampam) [inline README.md:62]: Minimal required version for `avto-dev/identity-laravel` in `composer.json` must be updated

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eldario) [reply README.md:201]: It is valid, but it may not exist

File: README.md (hunk 3 above)
Comment (eldario) [reply README.md:62]: `changed`

File: src/Frameworks/Laravel/ServiceProvider.php (hunk 2 above)
Comment (eldario) [reply src/Frameworks/Laravel/ServiceProvider.php:111]: `added` a row in `ServiceProviderTest`

**Review by tarampampam — APPROVED** [review]

**Comment by tarampampam** [comment]
Good job, dude!

**Comment by eldario** [comment]
Great! Thanx for my PR.

**Pull request merged**